docs/resources/azure_sobject.md

```diff
@@ -121,14 +137,20 @@ resource "dsm_azure_sobject" "sobject" {
 | -------- | -------- |-------- |
 | `RSA` | 2048, 3072, 4096 | APPMANAGEABLE, SIGN, VERIFY, ENCRYPT, DECRYPT, WRAPKEY, UNWRAPKEY, EXPORT |
 | `EC` | NistP256, NistP384, NistP521,SecP256K1 | APPMANAGEABLE, SIGN, VERIFY, AGREEKEY, EXPORT
-- `key_size` (Number) The size of the security object.
-- `obj_type` (String) The type of security object.
+- `purge_deleted_key` (Boolean) Purge deleted key in Azure key vault.purging the key makes all data encrypted with it unrecoverable unless you later import the same key material from Fortanix DSM into the Azure key.The DSM source key is not affected by this operation. The supported values are true/false.
```

Comment:
`...into the Azure key.The` -> `...into the Azure Key Vault. The`

Reply:
Changed